config/nginx/conf.d/cantusdb.conf.production

```diff
@@ -1,6 +1,9 @@
 server {
+    # Redirect all http traffic to corresponding https page
```

Comment:
Returning  301 $host will return a redirect to the https version of the hostnames, which will then do a 301 to the canonical. You should just do cantusdatabase.org instead.

Reply:
Just change

`return 301 https://$host$request_uri;`

to

`return 301 https://cantusdatabase.org$request_uri;`

Reply:
great

Reply:
I can see why it was initially set up as it is - the first redirect is like a function that does just one thing (http -> https) and the second does just one other thing (variant subdomain -> canonical). But it does create extra network traffic, so I see the value in setting it up as you suggested, @ahankinson. Thanks for this suggestion - I just opened #1322 implementing it.